ApplyToBecomeInternal/ApplyToBecomeInternal/Views/TaskList/Index.cshtml

```diff
@@ -133,7 +133,7 @@
 			Preview and generate HTB template
 		</h2>
 		<ol class="app-task-list govuk-!-margin-bottom-8">
-			<a href="#" class="govuk-button govuk-!-margin-bottom-2">Preview HTB template</a>
+			<a href="/task-list/@Model.Id/preview-htb-template" class="govuk-button govuk-!-margin-bottom-2">Preview HTB template</a>
```

Comment:
Other areas of the project are using ASP.NET Tag Helpers, we should probably commit to one approach.